infra/main.json

```diff
@@ -4,8 +4,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.31.34.60546",
-      "templateHash": "9243131736696562084"
+      "version": "0.29.47.4906",
```

Comment:
Why have we downgraded the version of bicep - this should have only ever gone up. This makes me feel that bicep has not been updated on your local machine.

Reply:
Thanks for identifying @ross-p-smith . Its fixed now.